src/libYARP_dev/include/yarp/dev/IVisualServoing.h

```diff
@@ -130,6 +130,28 @@ class yarp::dev::IVisualServoing
     virtual bool setModality(const std::string& mode) = 0;
 
     /*!
+     *  Set visual servo control law between:
+     *  1. 'decoupled': image-based visual servoing with decoupled position and
+     *                  orientation control law, the control law was proposed
+     *                  in [1];
+     *  2. 'robust': image-based visual servoing with averaged image Jacobians,
+     *               the control law was proposed in [2];
+     *
+     * \param mode a label referring to one of the three visual servo controls,
+     *             i.e. 'position', 'orientation' or 'pose'.
+     *
+     * \note [1] C. Fantacci, G. Vezzani, U. Pattacini, V. Tikhanoff and L.
+     *       Natale, "Precise markerless visual servoing on unknown objects for
+     *       humanoid robot platforms", to appear.
+     *       [2] E. Malis, “Improving vision-based control using efficient
+     *       second-order minimization techniques”, IEEE ICRA, vol. 2, p.
+     *       1843–1848, 2004.
+     *
+     * \return true/false on success/failure.
+     */
```

Comment:
This description seems quite specific for a certain implementation of the Visual Servo control. Perhaps in the YARP interface the type of control law can be simply defined as "implementation-specific" and the concrete details can be contained in the implementation of the interface?

Reply:
I agree.
References to paper and other implementation details should go elsewhere, keeping the API as much agnostic as possible.

Reply:
That is actually true. Good catch!
I'll have a close look at the whole interface documentation keeping this viewpoint in mind and do a PR asap.

Thanks!